pandas/core/frame.py

```diff
@@ -1678,9 +1678,10 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
         fname : str
             string file path
         engine : {'auto', 'pyarrow', 'fastparquet'}, default 'auto'
-            Parquet reader library to use. If 'auto', then the option
-            'io.parquet.engine' is used. If 'auto', then the first
-            library to be installed is used.
+            Parquet library to use. If 'auto', the value from
+            ``io.parquet.engine`` is used. The default ``io.parquet.engine``
```

Comment:
I would clarify "io.parquet.engine", as now it seems like a constant in pandas.io.parquet.engine, which is not the case, it is an option. Maybe leaving "*the option* io.parquet.engine .." as before is enough